yew-ui/src/components/attendants.rs

```diff
@@ -208,14 +255,88 @@ impl Component for AttendantsComponent {
                     true
                 }
             },
-            Msg::OnPeerAdded(_email) => true,
+            Msg::OnPeerAdded(_email) => {
+                log!("New peer arrived.");
+                if self.e2ee_enabled {
+                    self.send_public_key(ctx);
+                }
+                true
+            }
             Msg::OnFirstFrame((_email, media_type)) => match media_type {
                 MediaType::SCREEN => true,
                 _ => false,
             },
             Msg::OnInboundMedia(response) => {
-                if let Err(e) = self.peer_decode_manager.decode(response) {
-                    log!("error decoding packet: {:?}", e);
+                match response.packet_type.enum_value() {
+                    Ok(PacketType::AES_KEY) => {
+                        if !self.e2ee_enabled {
+                            return false;
+                        }
+                        log!("Received AES_KEY", &response.email);
```

Comment:
Is there a way to set a log level like on the backend? this is great for debugging and I do want to keep it around, I would just like to hide it in prod

Reply:
Yes I can update this MR with that